.github/workflows/build_ubuntu_on_push.yml

```diff
@@ -0,0 +1,93 @@
+name: Build Ubuntu on push no testing
+
+on: 
+  push:
+  
+  workflow_dispatch:
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Info
+      run: echo GITHUB_WORKSPACE=$GITHUB_WORKSPACE
+      
+    - name: Apt update
+      run: sudo apt-get update
+
+    - name: Install tools
+      run: sudo apt-get -y install git cmake build-essential gettext help2man openssl wget #clang-3.8
+
+    #- name: Install gtest
+    #  run: sudo apt-get -y install libgtest-dev
+ 
+    #- name: Install gmock
+    #  run: sudo apt-get install -y libgmock-dev
+ 
+    - name: Prepare sources for act (run actions local)
+      run: echo #rm -r 3rdparty/aubio 3rdparty/ced
+
+    - name: Checkout 
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Prepare Submodules
+      run : git submodule update --init --recursive
+
+    - name: Install libraries
+      run: |
+        #sudo apt-get -y install software-properties-common
```

Comment:
I don't really understand why you are creating a build we already have. Would it make more sense to pull down one of the existing Ubuntu builds? All of the deps installed here are already installed in the Ubuntu containers so we don't waste time re-installing everything. If this does need to be built separately, then this should probably be run inside one of the Containers.

Reply:
@ooshlablu can you add a workflow that installes google test so that this one willl run the tests? The demo workflows aren't run here, cause they are startet if commits are pushed. I will remove them afterwards.

Reply:
I have done that already.